### PR TITLE
Notify admin

### PR DESCRIPTION
**Pull request by ThibautGery**
if the config `send_notify_email` is set to `true` then the signup of a new key send a mail to the administrator of Umbrella

**NOTE** : 
 * I wasn't able to run the application and check the mail. Nonetheless the tests are green.
 * I don't know Rails so feedback is welcome

### CODE REVIEW

**Comment by GUI** [comment]
On a quick glance, this looks great! The test coverage also looks great, so thank for doing that. I'll look to get this merged into master tomorrow. Thanks for the contribution!

File: src/api-umbrella/web-app/app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -49,20 +49,25 @@ def create
 
     respond_to do |format|
       if(@api_user.save)
-        send_email = (params[:options] && params[:options][:send_welcome_email].to_s == "true")
+        send_welcome_email = (params[:options] && params[:options][:send_welcome_email].to_s == "true")
+        send_notify_email = (params[:options] && params[:options][:send_notify_email].to_s == "true")
```
Comment (ThibautGery) [inline src/api-umbrella/web-app/app/controllers/api/v1/users_controller.rb]: Is there a specific code to write to get data from `ApiUmbrellaConfig[:send_notify_email]`?

**Comment by GUI** [comment]
Awesome, thanks for adding this feature!

**Comment by GUI** [comment]
Quick heads up: v0.12.0 packages have been [released](https://groups.google.com/forum/#!topic/api-umbrella/x5GfWdnho1M), which includes this update.

**Pull request merged**